bundles/azure/src/main/java/org/apache/gravitino/abs/fs/AzureFileSystemProvider.java

```diff
@@ -64,6 +74,42 @@ public FileSystem getFileSystem(@Nonnull Path path, @Nonnull Map<String, String>
 
     hadoopConfMap.forEach(configuration::set);
 
+    // Check whether this is from GVFS client.
+    if (config.containsKey(GravitinoVirtualFileSystemConfiguration.FS_GRAVITINO_SERVER_URI_KEY)) {
```

Comment:
why adding these checks?

Reply:
To make it clear that the call path is from GVFS client NOT Gravitino server.

Reply:
it's hard to understand the code,  I prefer to get credentials from Gravitino server first. add some configurations according to the credential type,  invoking `AzureSasCredentialProvider` seems hacky.

Reply:
I agreed, the configuration check here is really hard to understand, what's the relation between the check and the below code.